### DISCUSSION TITLE
Detect webpack configuration changes and rm -rf node_modules/.cache

### DISCUSSION THREAD

**Discussion by thgh** [payloadcms/payload#1408] [Feature Requests & Ideas]
Sounds possible and would improve DX quite a bit.

**Comment by jmikrut** [comment]
Totally. Actually we are about to dive into _**why**_ Webpack is not picking up these changes in the first place. Might be a simple bug. And / or might be solved by ditching Webpack altogether 😈 

But in any case I think this needs to be resolved ASAP. We're on it.

**Reply by thgh** [reply]
Do you lean towards esbuild / Vite / plain esmodules / turbopack?

**Reply by yorrd** [reply]
let's do esbuild <3 would solve all our payload-cms integration troubles

**Reply by onionhammer** [reply]
Please ditch webpack <3
